web/src/scripts/components/menu/FilterableList.jsx

```diff
@@ -138,6 +138,11 @@ export default class FilterableList extends Component {
 
     return _.orderBy(filteredList, [
       (pkg) => {
+        if (pkg.name) {
+          return pkg.name.split('/').length
```

Comment:
lol I think this is a great hack, but how is the performance ? Using RegEx might speed it up a little bit.